content/docs/command-reference/remote/modify.md

```diff
@@ -379,9 +384,12 @@ more information.
   not set.
 
   ```dvc
-  $ dvc remote modify myremote projectname myproject
+  $ dvc remote modify --local myremote projectname myproject
```

Comment:
I think we a bit to far. Let's only do actual private info - like passwords, etc.

Reply:
Oops. Rolled this one back.